frontend/src/components/Common/ReportDetailTableContainsSubtitle/index.tsx

```diff
@@ -114,15 +145,13 @@ export const ReportForThreeColumns = ({
   );
 
   return (
-    <>
-      <Container>
-        <ReportSelectionTitle>{title}</ReportSelectionTitle>
-        {errorMessage && <ErrorMessagePrompt errorMessage={errorMessage} style={{ marginBottom: '1.5rem' }} />}
-        {renderLoading()}
-        {renderData()}
-      </Container>
-    </>
+    <Container>
+      <ReportSelectionTitle>{title}</ReportSelectionTitle>
+      {errorMessage && <ErrorMessagePrompt errorMessage={errorMessage} style={{ marginBottom: '1.5rem' }} />}
```

Comment:
Can we move the style of this component into the <ErrorMessagePrompt>?

Reply:
maybe it's difficult because the style is delivered to ErrorMessagePrompt component and this is to allow components to have custom styles.